recipe/__init__.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+from __future__ import print_function, absolute_import
+
+__version__ = "0.11.7"
```

Comment:
Note it is run on Unix and [Windows]( https://github.com/conda-forge/ruamel_yaml-feedstock/pull/3/commits/7dd5b2675165f9197fae044eaa82333b77e1e3ae#diff-d04c86b6bb20341f5f7c53165501a393R8 ).